scripts/bash/backendStartDev.sh

```diff
@@ -26,10 +26,14 @@ while [ "$2" != "" ]; do
         type_script=true
         shift
         ;;
-    -s | --skip-internal)
+    -i | --include-internal)
         skip_internal=true
```

Comment:
I think you need to rename this to `inclue_internal`?

Reply:
Eeek, thanks for the review!! 👀